addons/test_mass_mailing/tests/test_mailing.py

```diff
@@ -320,8 +320,8 @@ def test_mailing_w_seenlist_unstored_partner(self):
     @mute_logger('odoo.addons.mail.models.mail_mail')
     def test_mailing_mailing_list_optout(self):
         """ Test mailing list model specific optout behavior """
-        mailing_contact_1 = self.env['mailing.contact'].create({'name': 'test 1A', 'email': 'test@test.example.com'})
-        mailing_contact_2 = self.env['mailing.contact'].create({'name': 'test 1B', 'email': 'test@test.example.com'})
+        mailing_contact_1 = self.env['mailing.contact'].create({'name': 'test 1', 'email': 'test@test.example.com'})
+        mailing_contact_2 = self.env['mailing.contact'].create({'name': 'test 1', 'email': 'test@test.example.com'})  # will create perfect duplicate -> not sent
```

Comment:
Not sure to understand this change ?

Reply:
Ah ok ^^ the "perfect duplicate" comment is maybe not crystal clear, we could say something like 'duplicate now checks for subject / body (/ attachments)' ?